src/crypto-api.ts

```diff
@@ -260,3 +269,17 @@ export class DeviceVerificationStatus {
         return this.localVerified || (this.trustCrossSignedDevices && this.crossSigningVerified);
     }
 }
+
+export interface ImportOpts {
+    stage: string; // TODO: Enum
+    successes: number;
+    failures: number;
+    total: number;
+}
+
+export interface ImportRoomKeysOpts {
+    /** called with an object that has a "stage" param */
```

Comment:
well it's called with an `ImportOpts`; that much is obvious. A more useful doc-comment here would be to say *when* it is called.

Reply:
Updarted